scraper.rb

```diff
@@ -41,7 +41,7 @@ def scrape_party(url)
       party: tds[2].text.strip,
       party_id: CGI.parse(URI.parse(url).query)['party'].first.gsub(/[{}]/,''),
       constituency: memberships.first[/ in (.*?) from/, 1].sub('greater constituency','').strip,
-      email: box.css('div.person a[href*="mailto:"]/@href').text.gsub('mailto:',''),
+      email: box.css('div.person a[href*="mailto:"]/@href').text.gsub('mailto:','').gsub('|',';').gsub(' / ',';'),
```

Comment:
I don't think you need two passes, or for them to be differently formatted like this — I believe we'll do the right thing later with leading or training spaces, so you may as well just handle "/" rather than only ones with spaces around them. And when substituting single characters like this, `tr` is probably better: i.e. you should just need `.tr('|/',';')`